src/nIsForName.js

```diff
@@ -22,7 +22,13 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  const letterSch = 'N';
+  const resName = names.filter(flName => {
```

Comment:
don't shorten words randomly. Whn u do ths cd is vr hrd to rd (When you do this code is very hard to read).

flName => fullName.